packages/scripts/build-package.ts

```diff
@@ -55,6 +59,15 @@ for (const rel of assets) {
   await cp(join("src", rel), join("lib/cjs", rel));
 }
 
+if (
+  await access("./src/_generated")
+    .then(() => true)
+    .catch(() => false)
+) {
+  await cp("./src/_generated", "./lib/_generated", { recursive: true });
+  await cp("./src/_generated", "./lib/cjs/_generated", { recursive: true });
+}
```

Comment:
a downside of this solution is that you end up with the generated client *twice* - one in the ESM and one in the CJS directory and that includes `libquery_engine-darwin-arm64.dylib.node` that weighs 14MB~.

I think it's a fair price to pay for this script to be decoupled from what actually happens with Prisma. This script, at the moment, doesn't know what it copies - it's just using a convention to copy over any `src/_generated` directory to dist files.